enterprise/web.core.syntax/src/org/netbeans/modules/web/core/syntax/ErrorAnnotationImpl.java

```diff
@@ -128,16 +129,13 @@ public void annotate(ErrorInfo[] errors){
         if (!added.isEmpty()) {
             final List finalAdded = added;
             final DataObject doJsp2 = doJsp;
-            Runnable docRenderer = new Runnable() {
-                @Override
-                public void run() {
-                    LineCookie cookie = (LineCookie)doJsp2.getCookie(LineCookie.class);
-                    Line.Set lines = cookie.getLineSet();
-
-                    for (Iterator i=finalAdded.iterator();i.hasNext();) {
-                        LineSetAnnotation ann=(LineSetAnnotation)i.next();
-                        ann.attachToLineSet(lines);
-                    }
+            Runnable docRenderer = () -> {
+                LineCookie cookie = (LineCookie)doJsp2.getCookie(LineCookie.class);
+                Line.Set lines = cookie.getLineSet();
+                
+                for (Iterator i=finalAdded.iterator();i.hasNext();) {
+                    LineSetAnnotation ann=(LineSetAnnotation)i.next();
+                    ann.attachToLineSet(lines);
                 }
             };
```

Comment:
`final List finalAdded = added;`
->
`final List<LineSetAnnotation> = added;`

after that you can change the for loop to:
```
for (LineSetAnnotation ann : finalAdded) {
    ann.attachToLineSet(lines);
}
```

Reply:
Agreed and done..